src/test/java/edu/harvard/hul/ois/fits/junit/DocMDTest.java

```diff
@@ -32,174 +28,82 @@
  *
  * @author dan179
  */
-public class DocMDTest extends AbstractLoggingTest {
-
-    /*
-     *  Only one Fits instance is needed to run all tests.
-     *  This also speeds up the tests.
-     */
-    private static Fits fits;
-
-    @BeforeClass
-    public static void beforeClass() throws Exception {
-        // Set up FITS for entire class.
-        File fitsConfigFile = new File("testfiles/properties/fits-full-with-tool-output.xml");
-        fits = new Fits(null, fitsConfigFile);
-    }
-
-    @AfterClass
-    public static void afterClass() {
-        fits = null;
-    }
+@Ignore("These tests have no asserts and just generate output files")
```

Comment:
Is there any expectation that these "tests" are ever to be run?

Reply:
I'm not entirely sure what the reasoning behind these "tests" is. My guess is that they were just used as a convenience to generate output for the files. Most, if not all, of the files that were having output generated for without assertions also have corresponding XML unit tests that actually do something.